File: HotDesk_PRAI/App_Module/ConfirmPassword.ascx.vb
Partial Class App_Module_ConfirmPassword
    Inherits BaseUserControl
#Region "Properties"
    Public Property Password() As String
        Get
            Return txtPassword.Text
        End Get
        Set(ByVal value As String)
            txtPassword.Text = value
        End Set
    End Property

    Public Property ConfirmPassword() As String
        Get
            Return txtConPassword.Text
        End Get
        Set(ByVal value As String)
            txtConPassword.Text = value
        End Set
    End Property
#End Region
#Region "Methods"
    Public Sub BindSetting()
        REM assign validation group to multiple controls
        txtConPassword.ValidationGroup = MyBase.ValidationGroup
        txtPassword.ValidationGroup = MyBase.ValidationGroup
    End Sub
#End Region
#Region "Events"
#Region "Overrides"
    Public Overrides Sub DataBind()
        Me.BindSetting()
        MyBase.DataBind()
    End Sub
#End Region
#End Region
End Class

File: TMS_HotDesk/App_Module/ConfirmPassword.ascx.cs
public partial class App_Module_ConfirmPassword : BaseUserControl
{
    #region Properties
    public string Password
    {
        get { return txtPassword.Text; }
        set { txtPassword.Text = value; }
    }

    public string ConfirmPassword
    {
        get { return txtConPassword.Text; }
        set { txtConPassword.Text = value; }
    }
    #endregion

    #region Methods
    public void BindSetting()
    {
        txtConPassword.ValidationGroup = base.ValidationGroup;
        txtPassword.ValidationGroup = base.ValidationGroup;
    }
    #endregion

    public override void DataBind()
    {
        this.BindSetting();
        base.DataBind();
    }
}
